gcloud-java-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java

```diff
@@ -1156,6 +1156,36 @@ public void testSignUrl() throws NoSuchAlgorithmException, InvalidKeyException,
   }
 
   @Test
+  public void testSignUrlLeadingSlash() throws NoSuchAlgorithmException, InvalidKeyException,
+      SignatureException, UnsupportedEncodingException {
+    String blobName = "/b1";
+    EasyMock.replay(storageRpcMock);
+    ServiceAccountAuthCredentials authCredentials =
+        ServiceAccountAuthCredentials.createFor(ACCOUNT, privateKey);
+    storage = options.toBuilder().authCredentials(authCredentials).build().service();
+    URL url = storage.signUrl(BlobInfo.builder(BUCKET_NAME1, blobName).build(), 14, TimeUnit.DAYS);
+    String stringUrl = url.toString();
+    String expectedUrl =
+        new StringBuilder("https://storage.googleapis.com/").append(BUCKET_NAME1).append(blobName)
+            .append("?GoogleAccessId=").append(ACCOUNT).append("&Expires=").append(42L + 1209600)
+            .append("&Signature=").toString();
+    System.out.println(stringUrl);
+    System.out.println(expectedUrl);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://www.codacy.com/app/mziccard/gcloud-java/file/2579566087/issues/source?bid=3123103&fileBranchId=3302344#l1173)